jsep.go

```diff
@@ -195,6 +195,9 @@ func (d *MediaDescription) WithExtMap(e ExtMap) *MediaDescription {
 
 // WithTransportCCExtMap adds an extmap to the media description
 func (d *MediaDescription) WithTransportCCExtMap() *MediaDescription {
+	extMapURI := map[int]string{
+		ExtMapValueTransportCC: "http://www.ietf.org/id/draft-holmer-rmcat-transport-wide-cc-extensions-01",
+	}
```

Comment:
Make extMapURI a non-global variable to pass "no global variable". extMapURI seems to be not exported or used anywhere?